docker-storage-setup.sh

```diff
@@ -346,11 +346,58 @@ create_lvm_thin_pool () {
   lvconvert -y --zero n $CHUNK_SIZE_ARG --thinpool $VG/$DATA_LV_NAME --poolmetadata $VG/$META_LV_NAME
 }
 
+get_configured_thin_pool() {
+  local options tpool opt
+
+  if [ ! -f "$DOCKER_STORAGE" ];then
```

Comment:
done. In fact I moved get_docker_storage_options() call into setup_storage() so that we call it once and then multiple functions can look at existing storage options.